Dockerfile

```diff
@@ -118,7 +120,8 @@ RUN apt-get update && \
         zlib1g \
         v4l-utils \
         libv4l-0 \
-        alsa-utils
+        alsa-utils \
+        gettext-base
```

Comment:
required for `envsubst` env variable token replacement

Reply:
Maybe there's a solution with `sed` instead of pulling in the gettext-base package.